### ISSUE TITLE
Busy with printing to vcf...

### ISSUE THREAD

**Issue by ndierckx** [mroosmalen/nanosv#62]
Hi,

Is it normal that it is almost 2 days stuck on "Busy with printing to vcf..."?

**Comment by jaesvi** [comment]
Not really, it should not be the most computationally intense step.  But of course it depends on your sample and coverage, and the number of breakpoints expected.

**Comment by ndierckx** [comment]
I don't think virtual memory is a problem and everything progresses relatively fast, but it gets stuck every time at "Busy with printing to vcf..." without any errors

**Comment by ndierckx** [comment]
Anything wrong with the command I use?
NanoSV -s samtools -b file.bed -o file.vcf file.bam

**Comment by Amanda2018genetics** [comment]
> I don't think virtual memory is a problem and everything progresses relatively fast, but it gets stuck every time at "Busy with printing to vcf..." without any errors

Hi, I was stuck in 'busy with printing to vcf ..." for more than 12 hours. Is that normal ?  And how did you  solve the  problem ?
thanks.

**Comment by ndierckx** [comment]
Hi, sorry forgot to answer. Never really resolved the problem, but sometimes after waiting a very long time, it did finish...
